File: 프로그래머스/2/42578. 의상/의상.java
import java.util.*;

class Solution {
    public int solution(String[][] clothes) {
        int answer = 1;
        Map<String, Integer> c = new HashMap<>();
        
        for(String[] clothe : clothes){
            c.merge(clothe[1], 1, Integer::sum);
        }
        
        for(Integer v : c.values()){
            answer *= v + 1;
        }
        
        return answer - 1;
    }
}

File: 프로그래머스/2/42578. 의상/의상.py
def solution(clothes):
    answer = 1
    
    c = {}
    
    for clothe in clothes:
        c[clothe[1]] = c.get(clothe[1], 0) + 1
    
    for value in c.values():
        answer *= value + 1
    
    return answer - 1